packages/nimble-components/src/rich-text/editor/index.ts

```diff
@@ -109,6 +114,9 @@ export class RichTextEditor extends FoundationElement implements ErrorPattern {
         // Tiptap [isEmpty](https://tiptap.dev/api/editor#is-empty) returns false even if the editor has only whitespace.
         // However, the expectation is to return true if the editor is empty or contains only whitespace.
         // Hence, by retrieving the current text content using Tiptap state docs and then trimming the string to determine whether it is empty or not.
+        if (this.tiptapEditor.state.doc.toString().includes('mention')) {
+            return this.tiptapEditor.isEmpty;
+        }
         return this.tiptapEditor.state.doc.textContent.trim().length === 0;
```

Comment:
I think this will not work, as before `this.tiptapEditor.isEmpty;` will return true even if there is only white spaces, So to overcome that we went for `this.tiptapEditor.state.doc.textContent.trim().length === 0;`, which will trim white spaces. But in mention case, If only mention tags are available inside the editor `this.tiptapEditor.state.doc.textContent.trim().length === 0;` is returning empty as true by ignoring mention tags. So to achieve this case , added the if block which will simply return empty based on the existence of mention tag in it.

Reply:
> by adding `|| this.tiptapEditor.state.doc.textContent.trim().length === 0`.

Actually for the mention tags text content are considered as two white spaces, So the above text content will have only two white spaces in case of mention tags, If we are conditioning with `OR` it will return true as the `textcontent.trim().length` will return empty. 

> isEmpty actually returns true

`isEmpty` will return false if there are any mention nodes inside. So we are checking the condition of mention node existence in the editor document as `isEmpty` will return false for the mention nodes.